docs/addons.md

```diff
@@ -0,0 +1,240 @@
+
+Below you see an example giving **cluster-admin access** to the dashboard.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: kubernetes-dashboard
+  labels:
+    k8s-app: kubernetes-dashboard
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: kubernetes-dashboard
+  namespace: kube-system
+```
+
+### Monitoring with Heapster - Standalone
+
+**This addons is deprecated. Please use metrics-server instead**
+
+Monitoring supports the horizontal pod autoscaler.
+
+Install using:
+```
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/monitoring-standalone/v1.11.0.yaml
+```
+Please note that [heapster is retired](https://github.com/kubernetes/heapster/blob/master/docs/deprecation.md). Consider using [metrics-server](https://github.com/kubernetes-incubator/metrics-server) and a third party metrics pipeline to gather Prometheus-format metrics instead.
+
+### Monitoring with Prometheus Operator + kube-prometheus
+
+The [Prometheus Operator](https://github.com/coreos/prometheus-operator/) makes the Prometheus configuration Kubernetes native and manages and operates Prometheus and Alertmanager clusters. It is a piece of the puzzle regarding full end-to-end monitoring.
+
+[kube-prometheus](https://github.com/coreos/prometheus-operator/blob/master/contrib/kube-prometheus) combines the Prometheus Operator with a collection of manifests to help getting started with monitoring Kubernetes itself and applications running on top of it.
+
+```console
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/prometheus-operator/v0.26.0.yaml
+```
+
+### Route53 Mapper
+
+**This addon is deprecated. Please use [external-dns](https://github.com/kubernetes-sigs/external-dns) instead.**
+
+Please note that kOps installs a Route53 DNS controller automatically (it is required for cluster discovery).
+The functionality of the route53-mapper overlaps with the dns-controller, but some users will prefer to
+use one or the other.
+[README for the included dns-controller](https://github.com/kubernetes/kops/blob/master/dns-controller/README.md)
+
+route53-mapper automates creation and updating of entries on Route53 with `A` records pointing
+to ELB-backed `LoadBalancer` services created by Kubernetes. Install using:
+
+The project is created by wearemolecule, and maintained at
+[wearemolecule/route53-kubernetes](https://github.com/wearemolecule/route53-kubernetes).
+[Usage instructions](https://github.com/kubernetes/kops/blob/master/addons/route53-mapper/README.md)
+
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/route53-mapper/v1.3.0.yml
+```
+
+### Custom addons
+
+The docs about the [addon management](/development/addons.md#addon-management) describe in more detail how to define a addon resource with regards to versioning.
```

Comment:
This link doesn't work on the live preview site (check [here](https://deploy-preview-10355--kubernetes-kops.netlify.app/addons/#custom-addons))

I don't know for sure that this will work but its worth a shot:
```suggestion
The docs about the [addon management](../development/addons#addon-management) describe in more detail how to define a addon resource with regards to versioning.
```

Reply:
Thanks for catching. Removing the slash worked.